problems/objects.js

```diff
@@ -226,16 +353,24 @@ const updateList = () => {};
  * @returns {Object} The Object without the key.
  */
 
- const deleteKey = () => {};
-
+const deleteKey = (object, key) => {
+  delete object[key]
+  return object
+};
 
- /**
-  * Takes in an object and returns the number of
-  * properties it has.
-  * @param {Object} obj
-  * @returns {number} Number of properties.
-  */
- const propertyCount = () => {};
+/**
+ * Takes in an object and returns the number of
+ * properties it has.
+ * @param {Object} obj
+ * @returns {number} Number of properties.
+ */
+const propertyCount = (obj) => {
+  let counter = 0
```

Comment:
We can get all the properties of an object through Object.values() and then use .length() to find the number of properties. Try refactoring this function to be one line.